partner_contact_age_range/__init__.py

```diff
@@ -0,0 +1,3 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
```

Comment:
Make all license lines https:

Reply:
This still needs to be taken up. Then we are good to go.

Reply:
> This still needs to be taken up. Then we are good to go.

@NL66278 done